banzai/utils/file_utils.py

```diff
@@ -46,9 +46,20 @@ def instantly_public(proposal_id):
     return public_now
 
 
-def ccdsum_to_filename(image):
-    return 'bin{ccdsum}'.format(ccdsum=image.ccdsum.replace(' ', 'x'))
+def ccdsum_to_filename(ccdsum):
```

Comment:
Is there a particular reason to be inconsistent in the function parameters? It is an image object for filter_to_filename and config_to_filename, but the actual value for ccdsum_to_filename

Reply:
It is called inside the image constructor so I had originally changed it, but doesn't need to be. Fixed.